great_expectations/rule_based_profiler/expectation_configuration_builder/default_expectation_configuration_builder.py

```diff
@@ -236,7 +236,7 @@ def _build_binary_list(
 
     def _build_boolean_result(
         self,
-        binary_list: List[Union[bool, str]],
+        binary_list: Union[ParseResults, List[Union[bool, str]]],
```

Comment:
Why this change? Can we possibly narrow down to a single type?

Reply:
@cdkini The previous version was getting PyCharm complaints.  @NathanFarmer if you think of a better way, please make the fix as you deem appropriate.  Thanks.